src/main/views/case-name-search.njk

```diff
@@ -0,0 +1,81 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "./macros/common-components.njk" import submitButton %}
+
+{% extends "template.njk" %}
+{% block pageTitle %}
+    {{ "Case Name Search" | safe }}
+{% endblock %}
+{% block beforeContent %}
+    {{ govukBackLink({
+        text: text,
+        href: '#',
+        attributes: {
+            onClick: "history.back();event.preventDefault();"
+        }
+    }) }}
+{% endblock %}
+{% block content %}
+    {% if noResultsError %}
+        {% set errorMessage = 'Case name not found. Please try again or search by case reference number, unique reference number, or by a whole court' %}

```

Comment:
Error message is different from the spec in the task description. Please see Ticket 6 section in the description.